notebooks/week6/01.send_request_to_endpoint.py

```
@@ -0,0 +1,135 @@
+
+# COMMAND ----------
+
+# MAGIC %pip install /Volumes/mlops_test/power_consumptions/packages/housing_price-0.0.1-py3-none-any.whl
+
+# COMMAND ----------
+
+# MAGIC %restart_python
+
+# COMMAND ----------
+
+import pandas as pd
```

Ruff (0.7.0)

`pandas` imported but unused

Remove unused import: `pandas`

(F401)